app/metrics/iq_activity.py

```diff
@@ -0,0 +1,208 @@
+"""Uses the filesystem to track various metrics about image-query activity. Tracks iqs, escalations,
+and audits for each detector, as well as iqs submitted to the edge-endpoint as a whole.
+
+Filesystem structure:
+/opt/groundlight/edge-metrics/
+    detectors/
+        <detector_id1>/
+            iqs
+            iqs_YYYY-MM-DD_HH   <--- Current hour
+            iqs_YYYY-MM-DD_HH   <--- Previous hour
+            iqs_YYYY-MM-DD_HH   <--- 2 hours ago
+            escalations
+            escalations_YYYY-MM-DD_HH   <--- Current hour
+            escalations_YYYY-MM-DD_HH   <--- Previous hour
+            escalations_YYYY-MM-DD_HH   <--- 2 hours ago
+            audits
+            audits_YYYY-MM-DD_HH   <--- Current hour
+            audits_YYYY-MM-DD_HH   <--- Previous hour
+            audits_YYYY-MM-DD_HH   <--- 2 hours ago
+        <detector_id2>/
+            repeat of above detector
+    iqs
+    iqs_YYYY-MM-DD_HH   <--- Current hour
+    iqs_YYYY-MM-DD_HH   <--- Previous hour
+    iqs_YYYY-MM-DD_HH   <--- 2 hours ago
+"""
+
+import logging
+import os
+from datetime import datetime, timedelta
+from functools import lru_cache
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+
+class FilesystemActivityTrackingHelper:
+    """Helper class to support tracking image-query activity using the filesystem.
+    This is just a skeleton and only supports timestamps right now.  But
+    we will expand this to support counting metrics, etc."""
```

Comment:
```suggestion
    """
```
You did that, rigth?